staging/src/k8s.io/kubectl/pkg/util/interrupt/interrupt.go

```diff
@@ -76,6 +77,8 @@ func (h *Handler) Signal(s os.Signal) {
 			fn()
 		}
 		if h.final == nil {
+			// Add new line before exiting to fix kubectl #1544
+			fmt.Print("\n")
```

Comment:
This is a breaking change, since we don't know which component (not only in the k8s codebase, but also external consumers) relies on this behavior. I'd recommend embracing a different approach other than making changes in here.

/hold

Reply:
Thanks for clarification, the problem is that we need to catch the termination signal and add new line, maybe we can create a new function for this behavior and trigger it in the upper class (for example: `get`) and enable it so under that condition would add new line.
Or if there is any other solution i would be happy to discuss about it.